standard_reports/item-usage/latest/convert_data_js/src/convert_data.ts

```diff
@@ -0,0 +1,92 @@
+import { ArrayElement, ConvertData } from '../convertDataType';
+import { Arguments } from './generated-types/arguments';
+import { ItemUsageQuery } from './generated-types/graphql';
+import get from 'lodash/get';
+import orderBy from 'lodash/orderBy';
+
+type SqlResult = { item_id: string; quantity: number }[];
```

Comment:
yeah quite a bit of type definitions, although i think it's worth it, would have saved time in initial dev i think